Sources/Networking/NetworkRequestStateController.swift

```diff
@@ -94,13 +94,15 @@ public final class NetworkRequestStateController {
     /// - Parameters:
     ///   - request: The request to send.
     ///   - requestBehaviors: Additional behaviors to append to the request.
-    public func send(request: any NetworkRequest, requestBehaviors: [RequestBehavior] = [], retryCount: Int = 2) {
+    ///   - retryCount: The number of times the action can be retried.
+    ///   - scheduler: The scheduler to receive the call on. The default value is `DispatchQueue.main`.
+    public func send(request: any NetworkRequest, requestBehaviors: [RequestBehavior] = [], retryCount: Int = 2, scheduler: some Scheduler = DispatchQueue.main) {
```

Comment:
For consistency with the other libraries can you add after the `scheduler: some Scheduler = DispatchQueue.main` after the request parameter.
```suggestion
    public func send(request: any NetworkRequest, scheduler: some Scheduler = DispatchQueue.main, requestBehaviors: [RequestBehavior] = [], retryCount: Int = 2) {
```